test/integration/default/controls/rabbitmq_users_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+service_name = 'rabbitmq-server'
+
+control 'rabbitmq users' do
+  describe command('rabbitmqctl list_users') do
+    its('stdout') { should match (/user1/) }
+    its('stdout') { should match (/user2/) }
+    its('stdout') { should_not match (/guest/) }
+  end
+end
```

Comment:
@sylvainfaivre Thanks for adding the InSpec test.  `rubocop --safe-auto-correct` is your friend, it will usually correct most of the lint issues.  While you can use `match`, that's usually only necessary when you're working with regex values.  So here's a fix up for you:

* [`rabbitmq_users_spec.rb`](https://github.com/myii/rabbitmq-formula/commit/57efa458af19851ae030eb788f35fcf20bb157b6#diff-8ebfbe554a46eaedcb24dc92793cb951)
* https://travis-ci.org/myii/rabbitmq-formula/jobs/615021857#L1673

Reply:
merged, thanks